libraries/CurieBLE/src/internal/BLECallbacks.cpp

```diff
@@ -223,4 +223,8 @@ void ble_central_device_found(const bt_addr_le_t *addr,
                                                   ad, len);
 }
 
+void ble_on_write_no_rsp_complete(struct bt_conn *conn, uint8_t err,
```

Comment:
I don't like that empty function neither.  Let me check it out more.  It should be made better.